src/olympia/addons/serializers.py

```diff
@@ -152,21 +154,27 @@ class AddonSerializer(serializers.ModelSerializer):
 
     class Meta:
         model = Addon
-        fields = ('id', 'authors', 'average_daily_users', 'current_version',
-                  'default_locale', 'description', 'edit_url', 'guid',
-                  'homepage', 'icon_url', 'is_disabled', 'is_experimental',
-                  'is_listed',
-                  'is_source_public', 'name', 'last_updated', 'previews',
-                  'public_stats', 'ratings', 'review_url', 'slug', 'status',
-                  'summary', 'support_email', 'support_url', 'tags',
-                  'theme_data', 'type', 'url', 'weekly_downloads')
+        fields = ('id', 'authors', 'average_daily_users', 'categories',
+                  'current_version', 'default_locale', 'description',
+                  'edit_url', 'guid', 'homepage', 'icon_url', 'is_disabled',
+                  'is_experimental', 'is_listed', 'is_source_public', 'name',
+                  'last_updated', 'previews', 'public_stats', 'ratings',
+                  'review_url', 'slug', 'status', 'summary', 'support_email',
+                  'support_url', 'tags', 'theme_data', 'type', 'url',
+                  'weekly_downloads')
 
     def to_representation(self, obj):
         data = super(AddonSerializer, self).to_representation(obj)
         if data['theme_data'] is None:
             data.pop('theme_data')
         return data
 
+    def get_categories(self, obj):
+        return {
+            app.short: [cat.slug for cat in obj.app_categories[app]]
```

Comment:
A comment or docstring in this function would be handy, just explaining what it is. I know one could trace back, but I think it would be handy just to explain what it is 😄